### PR TITLE
docs: links in readme and banner

### PR DESCRIPTION
**Pull request by mffap**
```[tasklist]
### Definition of Ready
- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,14 +68,22 @@ If you want to experience a hands-free ZITADEL, you should use [ZITADEL Cloud](h
...
+### SDKs
+
+Use our [SDKs](https://zitadel.com/docs/examples/sdks) for your favorite language and framework.
+
 ## Why choose ZITADEL
 
-We built ZITADEL with a complex multi-tenancy architecture in mind and provide the best solution to handle B2B customers and partners.
-Yet it offers everything you need for a customer identity (CIAM) use case.
+We built ZITADEL with a complex multi-tenancy architecture in mind and provide the best solution to handle [B2B customers and partners](https://zitadel.com/docs/guides/solution-scenarios/b2b).
+Yet it offers everything you need for a customer identity ([CIAM](https://zitadel.com/docs/guides/solution-scenarios/b2c)) use case.
```
Comment (fforootd) [inline README.md:82]:
```suggestion
Yet it offers everything you need for a customer identity [CIAM](https://zitadel.com/docs/guides/solution-scenarios/b2c) use case.
```

**Review by fforootd — COMMENTED** [review]
There might be a typo in the link

File: README.md (hunk 1 above)
Comment (mffap) [reply README.md:82]: ... customer identity (CIAM) use case ... 

I think for that is was fine. We can also just use CIAM.

Comment (mffap) [reply README.md:82]: ... customer identity (CIAM) use case ... 

I think for that is was fine. We can also just use CIAM.

**Review by fforootd — APPROVED** [review]

**Pull request merged**